src/commands/createNewProject/NewProjectLanguageStep.ts

```diff
@@ -45,16 +44,18 @@ export class NewProjectLanguageStep extends AzureWizardPromptStep<IProjectWizard
             { label: ProjectLanguage.JavaScript },
             { label: ProjectLanguage.TypeScript },
             { label: ProjectLanguage.CSharp },
+            { label: ProjectLanguage.PowerShell, description: previewDescription },
             { label: ProjectLanguage.Python, description: previewDescription },
             { label: ProjectLanguage.Java }
         ];
 
-        if (getFuncExtensionSetting('enablePowerShell')) {
-            languagePicks.push({ label: ProjectLanguage.PowerShell, description: previewDescription });
-        }
-
         const options: QuickPickOptions = { placeHolder: localize('selectFuncTemplate', 'Select a language for your function project') };
         wizardContext.language = <ProjectLanguage>(await ext.ui.showQuickPick(languagePicks, options)).label;
+
+        if (wizardContext.language === ProjectLanguage.PowerShell) {
```

Comment:
This is fine for now but we might want to refactor this into a "Languages support manangedDeps" array or something eventually.

Reply:
We are turning Managed Dependencies on by default, we no longer need this check nor the prompt.